docs/download-apis.js

```diff
@@ -8,47 +8,54 @@
  * Saves each model under `_doc-models/<version>`.
  */
 
-const chalk = require("chalk");
-const download = require("download");
-const fs = require("fs-extra");
-const path = require("path");
+import { createRequire } from "node:module";
+const require = createRequire(import.meta.url);
```

Comment:
I would just refactor the JSON import to use `node:fs` instead. It seems like a bad pattern to use require for this purpose, especially in an ESM package.

Reply:
Actually you already have fs-extra, so use `fse.readJson`.